frontend/device/android/device.lua

```diff
@@ -10,6 +10,13 @@ local T = require("ffi/util").template
 local function yes() return true end
 local function no() return false end
 
+local function canUpdateApk()
+    local flavor = android.getFlavor()
+    -- disable updates on fdroid builds, since they manage their own repo.
+    -- disable updates on debug builds too.
```

Comment:
Is that necessary? It seems like it just complicates testing without a clear gain.

Reply:
disable on debug builds? not necessary

Reply:
Well, I wouldn't then. :-)

Reply:
ok, but are you fine with the flavor of debug builds being "debug" or shall I change that too (and make the flavor settable, like on release builds)?

Reply:
Oh right, that should definitely be settable.

Reply:
Kinda defeats the point of a debug build otherwise, imho.